storage/blockdevice/COMPONENT_QSPIF/mbed_lib.json

```diff
@@ -37,15 +37,18 @@
         "MCU_NRF52840": {
             "QSPI_FREQ": "32000000",
             "QSPI_MIN_READ_SIZE": "4",
-            "QSPI_MIN_PROG_SIZE": "256"
+            "QSPI_MIN_PROG_SIZE": "4"
```

Comment:
@AGlass0fMilk FYI, I noticed this from your MCUboot port, and created this PR to systematically fix that 😉